ksql-engine/src/main/java/io/confluent/ksql/function/udf/datetime/TimestampToString.java

```diff
@@ -28,17 +28,23 @@
 
   @Override
   public Object evaluate(final Object... args) {
-    if (args.length != 2) {
-      throw new KsqlFunctionException("TimestampToString udf should have two input argument:"
-                                      + " date value and format.");
+    if (args.length < 2) {
+      throw new KsqlFunctionException("TimestampToString udf should have at least "
+                                    + "two input arguments: date value and format.");
+    }
+
+    if (args.length > 3) {
```

Comment:
The `KsqlFunctionException` thrown at the bottom of this method only shows the first two args - can we change it so that it includes all args, please?

Reply:
@big-andy-coates Ready 👍